modules/index.js

```diff
@@ -0,0 +1,11 @@
+import getDateTime from './datetime.js';
+import SavelocalItems from './localstorage.js';
+import ListBooks from './booklist.js';
+import navigation from './navigation.js';
+
+SavelocalItems();
+ListBooks();
+navigation();
+window.addEventListener('load', getDateTime);
+
+/* eslint-disable  max-classes-per-file */
```

Comment:
- [ ] Kindly remove the eslint disable. It is at the tail end of the file and disables nothing. Kindly remove since it is unnecessary - to keep our code clean.